docs/running-on-yarn.md

```diff
@@ -54,8 +54,8 @@ In `yarn-cluster` mode, the driver runs on a different machine than the client,
 
 # Preparations
 
-Running Spark-on-YARN requires a binary distribution of Spark which is built with YARN support.
-Binary distributions can be downloaded from the Spark project website. 
+Running Spark on YARN requires a binary distribution of Spark which is built with YARN support.
```

Comment:
See http://people.apache.org/~pwendell/spark-nightly/spark-master-docs/latest/running-on-yarn.html and search for `./bin/spark-submit --class path.to.your.Class`. It renders incorrectly since there's the indent + backticks that make the docs go awry. That's why I fixed that, too.

What do you propose as *a different approach going forward*? When I see a change without changes to the docs, what's the *proper* approach?